src/containers/EditorsColumn.js

```diff
@@ -53,4 +47,4 @@ function mapDispatchToProps(dispatch) {
 export default connect(
   mapStateToProps,
   mapDispatchToProps,
-)(EditorsColumn);
+)(resizableFlex(3)(EditorsColumn));
```

Comment:
This is very clear - allows for 2 resizable containers in the workspace component.